packages/suite/src/components/wallet/CoinjoinSummary/CryptoAmountWithHeader.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon, IconType, variables } from '@trezor/components';
+import { FiatValue } from '@suite-components/FiatValue';
+import { FormattedCryptoAmount } from '@suite-components/FormattedCryptoAmount';
+import { NetworkSymbol } from '@wallet-types';
+
+const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const Header = styled.div`
+    display: flex;
+    align-items: center;
+    font-size: ${variables.FONT_SIZE.TINY};
+    line-height: 1;
+`;
+
+const HeaderIcon = styled(Icon)`
+    margin-right: 4px;
+    padding-bottom: 2px;
+`;
+
+const CryptoAmount = styled(FormattedCryptoAmount)<{ color?: string }>`
+    margin: 6px 0 4px;
+    color: ${({ theme, color }) => color || theme.TYPE_DARK_GREY};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    font-size: ${variables.FONT_SIZE.H3};
+`;
+
+const Note = styled.p`
+    margin-top: 4px;
+`;
+
+interface CryptoAmountWithHeaderProps {
+    header: React.ReactNode;
+    headerIcon?: IconType;
+    value: string;
+    symbol: NetworkSymbol;
+    color?: string;
+    note?: React.ReactNode;
+    className?: string;
+}
+
+export const CryptoAmountWithHeader = ({
+    header,
+    headerIcon,
+    value,
+    symbol,
+    color,
+    note,
+    className,
+}: CryptoAmountWithHeaderProps) => (
+    <Container className={className}>
+        <Header>
+            {headerIcon && <HeaderIcon icon={headerIcon} size={12} />} {header}
+        </Header>
+
+        <CryptoAmount value={value} symbol={symbol} color={color} />
```

Comment:
Could this component also convert the amount to the correct unit? E.g. I provide `account.balance` and it converts it from `sat` to `BTC`.

Reply:
It is converting but accepts amounts in `sat` after recent changes by Marek, I will change it soon.

Reply:
@marekrjpolak could you confirm that you wanted as much as possible to use `sat`? Essentially, going forward, it's preferred to use `BTC` and like only for displaying for users?